### ISSUE TITLE
Let's release version 0.28.1

### ISSUE THREAD

**Issue by kevinbackhouse** [Exiv2/exiv2#2813]
There are have been a few updates on the [0.28.x](https://github.com/Exiv2/exiv2/commits/0.28.x) branch since we released [v0.28.0](https://github.com/Exiv2/exiv2/releases/tag/v0.28.0). I'd like to release version v0.28.1. Any objections to me going ahead with that? I'll update the 3 same files as #2602 did, and create a new tag.

**Comment by kevinbackhouse** [comment]
Output of `git shortlog -s -n v0.28.0..0.28.x`

```
    22  Miloš Komarčević
    13  Rosen Penev
     9  kamiccolo
     8  Ryan Friedman
     8  norbertwg
     5  Kevin Backhouse
     4  Daniel Vogelbacher
     2  Matthias Andree
     2  Pino Toscano
     2  Thomas Klausner
     1  Matthias Kuhn
     1  Mohammed Anas
     1  Peter Kovář
     1  luzpaz
     1  mergify[bot]
```

That's: @kmilos @neheb @kamiccolo @Ryanf55 @norbertwg @kevinbackhouse @cytrinox @mandree @pinotree @0-wiz-0 @m-kuhn @mhmdanas @1div0 @luzpaz

**Comment by kevinbackhouse** [comment]
Milestone for this release: https://github.com/Exiv2/exiv2/milestone/12

**Comment by kevinbackhouse** [comment]
This release will also fix [CVE-2023-44398](https://github.com/Exiv2/exiv2/security/advisories/GHSA-hrw9-ggg3-3r4r).

**Comment by neheb** [comment]
I'd like to get some of https://github.com/Exiv2/exiv2/pull/2712 in 0.28.x

**Comment by kmilos** [comment]
What about restoring wchar/wstring API for Windows? It's kinda important, but at the same time I feel we shouldn't hold off getting these fixes out? 0.28.2 or 0.29?

**Comment by kevinbackhouse** [comment]
> What about restoring wchar/wstring API for Windows? It's kinda important, but at the same time I feel we shouldn't hold off getting these fixes out? 0.28.2 or 0.29?

Sorry, I only just noticed this comment after I created the release. We could do 0.28.2 quite soon if you like.
